modules/backup/variables.tf

```diff
@@ -23,3 +23,53 @@ variable "s3_access_logs_bucket_name" {
   description = "Define name for the S3 access logs"
   type        = string
 }
+
+# KMS CMK
+
+variable "enable_s3_kms_key" {
+  description = "Enable creation of KMS key for S3 bucket encryption"
+  type        = bool
+  default     = false
+}
+
+variable "s3_key_tags" {
+  description = "A map of tags to assign to the resources."
+  type        = map(string)
+  default     = {}
+}
+
+variable "s3_key_rotation_enabled" {
+  description = "Specifies whether key rotation is enabled."
+  type        = bool
+  default     = true
+}
+
+variable "s3_cmk_alias" {
+  description = "The alias for the CMK key."
+  type        = string
+  default     = "graphdb-cmk-key"
+}
+
+variable "s3_key_enabled" {
+  description = "Specifies whether the key is enabled."
+  type        = bool
+  default     = true
+}
+
+variable "s3_key_spec" {
+  description = "Specification of the Key."
+  type        = string
+  default     = "SYMMETRIC_DEFAULT"
+}
+
+variable "s3_key_deletion_window_in_days" {
+  description = "The waiting period, specified in number of days for AWS to delete the KMS key(Between 7 and 30)."
+  type        = number
+  default     = 30
+}
+
+variable "s3_key_admin_arn" {
+  description = "ARN of the role or user granted administrative access to the SNS KMS key."
+  type        = string
+  default     = ""
```

Comment:
You don't need a default here as it comes from the variables.tf on root level

Reply:
Got it. Thanks